cmake/third_party/gcp_sdk.cmake

```diff
@@ -0,0 +1,73 @@
+cmake_minimum_required(VERSION 3.10)
+include(ExternalProject)
+include(GNUInstallDirs)
+include(${CMAKE_SOURCE_DIR}/cmake/helpers.cmake)
+
+# Skip the GCP SDK build step if the extension is not enabled.
+if(NOT ENABLE_GCP)
+    return()
+endif()
+
+config_choice(
+    IMPORT_GCP_SDK
+    "Specify how to import the GCP SDK"
+    OPTIONS
+        "none;IMPORT_GCP_SDK_NONE;NOT ENABLE_GCP"
+        "package;IMPORT_GCP_SDK_PACKAGE;ENABLE_GCP"
+        "external;IMPORT_GCP_SDK_EXTERNAL;ENABLE_GCP"
+)
+
+if(IMPORT_GCP_SDK_NONE)
+    message(FATAL_ERROR "Cannot enable GCP extension without specifying an IMPORT_GCP_SDK method (package, external).")
+endif()
+
+if(IMPORT_GCP_SDK_PACKAGE)
+    message(FATAL_ERROR "Importing GCP via package is not currently implemented.")
```

Comment:
Similar to above, suggest "Importing the GCP SDK via package is not yet implemented."

Is there a ticket to do this work? If so, it would be worth noting here.

Reply:
We are going to make another ticket for this.

Reply:
Sounds good! Create the ticket and add a comment with the ticket number. We have a standard `FIXME-WT-XXXX` syntax for this.

Reply:
Updated: 4e4158cb4eadfe8bde80cf2d1e55d11e91c217fb